__tests__/app.test.js

```diff
@@ -85,9 +85,10 @@ describe('GET /api/reviews/:review_id', () => {
         votes: 1,
         category: 'strategy',
         owner: 'tickle122',
-        created_at: '2021-01-18T10:00:20.514Z'
+        created_at: '2021-01-18T10:00:20.514Z',
+        comment_count: "3"
       }
-    expect(review[0]).toEqual(expected);
+    expect(review).toEqual(expect.objectContaining(expected));
     })
```

Comment:
Rather than editing the previous test, could we please see a new one for this ticket?